profiles/js/functions.js

```diff
@@ -235,10 +235,21 @@ async function FillData(uid, mode, completeReload = true) {
             document.getElementById("name__main").innerHTML = GetStringRawNonAsync("profiles", "profile.bar.user.new", [oData.username]);
         }
         if (oData.username != null) document.getElementById("name__sub").innerHTML = `
-    <img class="profiles__cover-country" src="https://osu.ppy.sh/images/flags/${oData.country_code}.png" id="country__flag"> ${oData.username} <div id="user__badges" class="profiles__user-badges"></div>
-    <a data-tippy-content="View profile on osu.ppy.sh" href="https://osu.ppy.sh/users/${oData.id}" target="_blank" class="profiles__cover-link"><i class="fas fa-external-link-alt"></i></a>`;
+        <div class="profiles__cover-info-name">
+            <img class="profiles__cover-country" src="https://osu.ppy.sh/images/flags/${oData.country_code}.png" id="country__flag">
+            <h1>${oData.username}</h1> 
+        </div>
+        <div id="user__badges" class="profiles__user-badges"></div>`;
+        document.getElementById("osu_link").href=  `https://osu.ppy.sh/users/${oData.id}`;

```

Comment:
super scuffed formatting here
```suggestion
        document.getElementById("osu_link").href = `https://osu.ppy.sh/users/${oData.id}`;

```